src/schema/index.js

```diff
@@ -30,11 +31,12 @@ const organizerSchema = Joi.object().keys({
 
 const alarmSchema = Joi.object().keys({
   action: Joi.string().regex(/audio|display|email/).required(),
-  trigger: Joi.array().required(),
+  trigger: Joi.any().required(),
   description: Joi.string(),
   duration: durationSchema,
   repeat: Joi.number(),
-  attach: Joi.string().uri(),
+  attach: Joi.string(),//To Support inbuilt iOS sounds
```

Comment:
Interesting. Would you give me an example of what arguments you pass into `alarmSchema` for iOS? If `.uri()` is too strict we will fix it.

Reply:
ATTACH;VALUE=URI:Glass

Works fine on Mac Os

Reply:
created in a new PR https://github.com/adamgibbons/ics/pull/59